package.json

```diff
@@ -4,20 +4,20 @@
   "description": "A boilerplate Node.js / Express app",
   "main": "index.js",
   "scripts": {
-    "start": "node app/index.js && npm run start-healthcheck",
-    "debug": "node --debug app/index.js",
+    "start": "babel-node app/index.js && npm run start-healthcheck",
```

Comment:
Here and further `node` was replaced with `babel-node` so we can get all es6 awesomeness on server side. This is especially cool since we are using universal modules.